crates/bindings/Cargo.toml

```diff
@@ -16,9 +16,9 @@ bench = false
 getrandom = ["spacetimedb-bindings-sys/getrandom"]
 
 [dependencies]
-spacetimedb-bindings-sys = { path = "../bindings-sys", version = "0.6.1" }
-spacetimedb-lib = { path = "../lib", default-features = false, version = "0.6.1"}
```

Comment:
cargo yelled at me about this one's use of `default-features` and I don't understand why, I'm investigating now because it sure does look like spacetimedb-lib has features.

Reply:
Ah I think I see. When we had multiple places specifying a dependency on the crate in lib/, they were unrelated as far as feature unification is concerned. But trying to share would make them have to agree, and since features can't go down, `default-features = false` is currently defined to be meaningless (but going to be an error in the future). So I think our options are either to just leave this one as a separately listed dependency on lib/, or to rework the feature set, or to have the workspace one turn default features off and all other uses turn it on.

Reply:
> or to have the workspace one turn default features off and all other uses turn it on.

Sounds the most sensible.